src/server/components/lists/find/handlers/translators-interpreters/languages.ts

```diff
@@ -0,0 +1,56 @@
+import type { Request, Response } from "express";
+import { languages } from "server/services/metadata";
+
+import { sanitiseLanguages } from "./../../helpers/sanitiseLanguages";
+import { getLanguagesRows } from "server/models/listItem/providers/helpers";
+import querystring from "querystring";
+
+export function get(req: Request, res: Response) {
+  const shouldReturn = req.query.return === "results";
+  const queryString = querystring.encode({ ...(shouldReturn && { return: "results" }) });
+
+  if (req.query.remove) {
+    const { languages: sessionLanguages = [] } = req.session.answers!;
+    const languagesAfterRemoving = sessionLanguages.filter((language) => language !== req.query.remove);
+
+    req.session.answers!.languages = languagesAfterRemoving;
+    req.session.answers!.languagesReadable = languagesAfterRemoving.map((lang) => languages[lang]);
+    res.redirect(`languages?${queryString}`);
+
+    return;
+  }
+  const languagesVM = Object.entries(languages).map(([value, text]) => ({ value, text }));
```

Comment:
I think for someone who isn't well versed on view-models maybe the name of this variable might bring some confusion. What are your thoughts on renaming it to something more descriptive. Like `languagesToArray` 🤷

Reply:
ViewModels are distinct, like how Models, Views and Controllers are. I want to convey "This is how the view will render the languages".

Reply:
I've changed so it's 

```
const languagesAsArray = splitLanguagesIntoArray();

//...

  const languagesViewModel = languagesAsArray;

```

Reply:
I think this works. Definitely more readable 👍🏿